nixos/modules/services/databases/mysql.nix

```diff
@@ -76,9 +78,64 @@ in
         description = "Location where MySQL stores its table files";
       };
 
+      configFile = mkOption {
+        type = types.path;
+        default = settingsFile;
+        defaultText = "settingsFile";
+        description = ''
+          Override the configuration file used by MySQL. By default,
+          NixOS generates one automatically.
```

Comment:
How about `By default, NixOS generates one automatically from <option>services.mysql.settings</option>`

Reply:
That is much more informative :+1: